xiaomi/aqara/remote_e1.py

```diff
@@ -0,0 +1,225 @@
+            },
+            3: {
+                # SizePrefixedSimpleDescriptor(
+                #   endpoint=3, profile=260, device_type=0,
+                #   input_clusters=["0x0012"], output_clusters=["0x0006"])
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: zha.DeviceType.ON_OFF_SWITCH,
+                INPUT_CLUSTERS: [MultistateInput.cluster_id],
+                OUTPUT_CLUSTERS: [OnOff.cluster_id],
+            },
+        },
+    }
+    replacement = {
+        ENDPOINTS: {
+            1: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: zha.DeviceType.ON_OFF_SWITCH,
+                INPUT_CLUSTERS: [
+                    BasicCluster,
+                    Identify.cluster_id,
+                    PowerConfigurationClusterH1Remote,
+                    MultistateInputCluster,
+                    AqaraRemoteManuSpecificCluster,
+                ],
+                OUTPUT_CLUSTERS: [
+                    Identify.cluster_id,
+                    OnOff.cluster_id,
+                    Ota.cluster_id,
+                ],
+            },
+            2: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: zha.DeviceType.ON_OFF_SWITCH,
+                INPUT_CLUSTERS: [
+                    MultistateInputCluster,
+                ],
+                OUTPUT_CLUSTERS: [
+                    OnOff.cluster_id,
+                ],
+            },
+            3: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: zha.DeviceType.ON_OFF_SWITCH,
+                INPUT_CLUSTERS: [
+                    MultistateInputCluster,
+                ],
+                OUTPUT_CLUSTERS: [
+                    OnOff.cluster_id,
+                ],
+            },
+        }
+    }
+    device_automation_triggers = {
+        # triggers when operation_mode == event
+        # the button doesn't send an release event after hold
+        (SHORT_PRESS, LEFT): {COMMAND: COMMAND_1_SINGLE},
+        (DOUBLE_PRESS, LEFT): {COMMAND: COMMAND_1_DOUBLE},
+        (TRIPLE_PRESS, LEFT): {COMMAND: COMMAND_1_TRIPLE},
+        (LONG_PRESS, LEFT): {COMMAND: COMMAND_1_HOLD},
+        (SHORT_PRESS, RIGHT): {COMMAND: COMMAND_2_SINGLE},
+        (DOUBLE_PRESS, RIGHT): {COMMAND: COMMAND_2_DOUBLE},
+        (TRIPLE_PRESS, RIGHT): {COMMAND: COMMAND_2_TRIPLE},
+        (LONG_PRESS, RIGHT): {COMMAND: COMMAND_2_HOLD},
+        (SHORT_PRESS, BOTH_BUTTONS): {COMMAND: COMMAND_3_SINGLE},
+        (DOUBLE_PRESS, BOTH_BUTTONS): {COMMAND: COMMAND_3_DOUBLE},
+        (TRIPLE_PRESS, BOTH_BUTTONS): {COMMAND: COMMAND_3_TRIPLE},
+        (LONG_PRESS, BOTH_BUTTONS): {COMMAND: COMMAND_3_HOLD},
+        # triggers when operation_mode == command
+        # known issue: it seems impossible to know which button being pressed
+        # when operation_mode == command
+        (ALT_SHORT_PRESS, BUTTON): {COMMAND: COMMAND_TOGGLE, ENDPOINT_ID: 1, ARGS: []},
```

Comment:
Are endpoint Id / args needed here?

Reply:
To be honest I do not know 🤷🏻‍♂️. I have only replicated the implementation of `remote_h1.py`:
* https://github.com/zigpy/zha-device-handlers/blob/dev/zhaquirks/xiaomi/aqara/remote_h1.py

Reply:
I have moved the file that was in a wrong path. 🤦🏻‍♂️ 
Removed estra args.